ansible/roles/mysql/tasks/main.yml

```diff
@@ -6,6 +6,7 @@
       name:
         - mariadb-server
         - firewalld
+      use_backend: dnf
```

Comment:
**TODO**: remove this file from the PR
+ remove the installation of dnf (in the bastion role maybe?)